<?xml version="1.0"?>
<launch>
	<group ns = "ipa_pars_map_analyzer">
		<!-- ipa_room_segmentation -->

		<!-- send parameters to parameter server -->
		<rosparam command="load" file="$(find ipa_room_segmentation)/ros/launch/room_segmentation_action_server_params.yaml" />
		<!-- external nodes to start -->
		<node pkg="ipa_room_segmentation" type="room_segmentation_server" name="room_segmentation_server" output="screen" respawn="true" respawn_delay="2" />

		<!-- nodes to start -->
		<node pkg="ipa_pars_map_analyzer" type="ipa_pars_map_analyzer_server" name="ipa_pars_map_analyzer_server" respawn="false" output="screen" />
		<node pkg="ipa_pars_map_analyzer" type="ipa_pars_map_tesselation_server" name="ipa_pars_map_tesselation_server" respawn="false" output="screen" />
		<node pkg="ipa_pars_map_analyzer" type="ipa_pars_map_knowledge_extractor_server" name="ipa_pars_map_knowledge_extractor_server" respawn="false" output="screen" />
		<node pkg="ipa_pars_map_analyzer" type="knowledge_to_yaml.py" name="ipa_pars_map_knowledge_to_yaml" respawn="false" output="screen" />

	</group>
</launch>
